examples/visual-tests/amp-story/amp-story-sidebar.html

```diff
@@ -25,6 +25,15 @@
         color: white;
         text-shadow: 3px 3px rgba(0, 0, 0, 0.5);
       }
+      amp-sidebar {
+        font-family: "Asap";
```

Comment:
Don't know if this is reliable or whether it's possible for it to flake between test runs; if it's not necessary, it might be best to leave it out

Reply:
Just the font family? Or the other styles here as well?

Reply:
Just the font family.  Our visual tests get screenshot diffs, and I don't know whether it will reliably load the font the same way (maybe it will and I'm just paranoid)